README.md

```diff
@@ -218,6 +226,124 @@ func userNotificationCenter(
 
 If your user taps on the notification this will be tracked back to Klaviyo as an "Opened Push" event assuming you have implemented the tracking changes discussed above.
 
+## Handling deep linking
+
+There are two use cases for deep linking that can be relevant here - 
+1. When you push a notification to your app with a deep link.
+2. Any other cases where you may want to deep link into your app via SMS, email, web browser etc.
+
+Note that Klaviyo doesn't officially yet support universal links but since there is no validation on the klaviyo front end for URI schemes, one can include universal links but making sure it works as per expectations is on the developer.
```

Comment:
I might reword this sliiiightly:

`Note that Klaviyo doesn't officially support universal links yet, but since there is no validation on the klaviyo front end for URI schemes, you can include universal links in your push notifications. Ensuring that Klaviyo push works to your expectations with universal links will be the responsibility of your developers.`

Reply:
Done.